reporter-web-app/src/components/TableView.js

```diff
@@ -20,17 +20,23 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import {
+    Button,
```

Comment:
Commit message: Why only to "several columns" and not all? How were the columns that got search added chosen? Please explain in the commit message.

Reply:
Updated the commit message - you can't have a column with search and drop-down so only added it to those without a drop-down